File: prak python/Clothing.py
from Product import Product

class Clothing(Product):

    size = ""
    material = ""
    gender = ""

    def __init__(self, size="", material="", gender=""):
        self.size = size
        self.material = material
        self.gender = gender

    def getSize(self):
        return self.size

    def setSize(self, size):
        self.size = size

    def getMaterial(self):
        return self.material

    def setMaterial(self, material):
        self.material = material

    def getGender(self):
        return self.gender

    def setGender(self, gender):
        self.gender = gender

File: prak c++/Clothing.cpp
#include "Product.cpp"
#include <string>

class Clothing : public Product {
private:
    std::string size;
    std::string material;
    std::string gender;

public:
    Clothing() : Product() {
        this->size = "";
        this->material = "";
        this->gender = "";
    }

    Clothing(std::string size, std::string material, std::string gender) : Product() {
        this->size = size;
        this->material = material;
        this->gender = gender;
    }

    std::string getSize() {
        return this->size;
    }

    void setSize(std::string size) {
        this->size = size;
    }

    std::string getMaterial() {
        return this->material;
    }

    void setMaterial(std::string material) {
        this->material = material;
    }

    std::string getGender() {
        return this->gender;
    }

    void setGender(std::string gender) {
        this->gender = gender;
    }
};